examples/typescript/lnd/src/nodeHandler.ts

```diff
@@ -0,0 +1,45 @@
+import { LnRpc, Peer, WalletUnlockerRpc } from '@radar/lnrpc';
+
+export default class NodeHandler {
+  constructor(private lnrpc: LnRpc & WalletUnlockerRpc) {}
```

Comment:
1. Dependency injection is nice for testing but I wonder whether, in such a small example, it really makes sense to use it here like the PR currently does.
2. With some abstraction, this app could work with multiple Lightning client implementations (`lnd`, `eclair`, etc.) but I'm torn on whether that's a good idea, or if it's better to have three separate but similar apps that use the different clients

Reply:
1. I think it's good to pass in the dependency
2. This would add a bunch of complexity for a basic example, so I think it's best to have separate apps for each LN implementation